### DISCUSSION TITLE
Add Julia language support to the analyzer

### DISCUSSION THREAD

**Discussion by Octogonapus** [aquasecurity/trivy#4437] [Ideas]
### Description

I would like to use Trivy to create SBOMs for Julia applications. I'd also like to analyze Julia packages for security problems, but that can come later on. Similarly to Rust, Julia has two TOML files which describe an application. These can be parsed and used to generate an SBOM.

### Target

SBOM

### Scanner

None

**Comment by Octogonapus** [comment]
The parser has been completed but needs more community support: https://github.com/aquasecurity/go-dep-parser/pull/219

**Comment by knqyf263** [comment]
Created https://github.com/aquasecurity/trivy/issues/5659
